pkg/chains/signing/kms/kms_test.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2023 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package kms creates a signer using a key management server
+
+package kms
+
+import (
+	"context"
+	"testing"
+
+	"github.com/tektoncd/chains/pkg/config"
+)
+
+func TestInValidVaultAddressTimeOut(t *testing.T) {
+	cfg := config.KMSSigner{}
+	cfg.Auth.Address = "http://test.com:8200"
+
+	_, err := NewSigner(context.TODO(), cfg)
+	expectedErrorMessage := "dial tcp 67.225.146.248:8200: i/o timeout"
```

Comment:
test.com might not always resolve to this IP in the future, how about using the IP address in `cfg.Auth.Address` instead of a hostname? we could also rely on a more well known IP like 8.8.8.8, wdyt?